src/event-stream/event-server.ts

```diff
@@ -728,105 +729,114 @@ export async function startEventServer(opts: {
       .json({ status: 'ready', msg: 'API event server listening for core-node POST messages' });
   });
 
-  app.post(
-    '*',
-    asyncHandler(async (req, res, next) => {
-      const eventPath = req.path;
-      let payload = JSON.stringify(req.body);
-      await messageHandler.handleRawEventRequest(eventPath, req.body, db);
-      if (logger.isDebugEnabled()) {
-        // Skip logging massive event payloads, this _should_ only exclude the genesis block payload which is ~80 MB.
-        if (payload.length > 10_000_000) {
-          payload = 'payload body too large for logging';
-        }
-        logger.debug(`[stacks-node event] ${eventPath} ${payload}`);
+  const handleRawEventRequest = asyncHandler(async req => {
+    await messageHandler.handleRawEventRequest(req.path, req.body, db);
+
+    const eventPath = req.path;
+    let payload = JSON.stringify(req.body);
```

Comment:
To save some CPU cycles, you should move these two lines to be inside the `logger.isDebugEnabled()` check

Reply:
done